packages/workers-shared/asset-worker/tests/handler.test.ts

```diff
@@ -224,4 +204,950 @@ describe("[Asset Worker] `handleRequest`", () => {
 		);
 		expect(response2.status).toBe(307);
 	});
+
+	describe("_headers", () => {
```

Comment:
I assume these are basically just cut-n-pasted from the pages-shared package?
Any significant differences?

Reply:
Workers Assets' asset-serving behavior is different with it's `canFetch` and `html_handling` etc. so just wanted to write some explicit tests ensuring we had behavior we were happy with for Workers Assets. These are all new tests written by me, and are very comprehensive, I think.

Reply:
Damn, I was hoping that I wouldn't need to look through these 😆